packages/hash/task-executor/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@hashintel/task-executor",
```

Comment:
The other workspaces have a `hash-` prefix after slash
```suggestion
  "name": "@hashintel/hash-task-executor",
```

Reply:
Will change for consistency :) I was a little unsure of if those were things specific to the HASH app (and this doesn't necessarily feel opinionated about it at the moment)

Reply:
Yeah I suppose it isn't tied to the HASH app in particular, but I think it will still be better to do this for consistency

Reply:
That's been done